installation/nginx.md

```diff
@@ -263,6 +297,10 @@ server {
 		proxy_set_header X-Real-IP              $remote_addr;
 		proxy_set_header X-Forwarded-For        $proxy_add_x_forwarded_for;
 		proxy_set_header X-Forwarded-Proto      $scheme;
+		satisfy                                 any;
+		allow                                   192.168.0.1/24;
+		allow                                   127.0.0.1;
+		deny                                    all;
```

Comment:
Yes sure, sorry for not being clear.
I meant if these four lines should be in the "Putting it all together" but yeah will not hurt I guess.